.github/workflows/staging-deploy.yml

```diff
@@ -3,7 +3,10 @@ name: Deploy to staging
 on:
   push:
     branches:
-      - "main"
+      - main
+    paths:
+      - 'frontend/**'
+      - 'frontend/.happy/**'
```

Comment:
Sorry, I marked my earlier comment as resolved. We don't need this, as it should be covered by the `frontend/**` line.